includes/Modules/Site_Verification.php

```diff
@@ -284,6 +311,15 @@ protected function setup_info() {
 		);
 	}
 
+	/**
+	 * Get the configured siteverification service instance.
+	 *
+	 * @return Google_Service|\Google_Service_SiteVerification
+	 */
+	private function get_siteverification_service() {
+		return $this->get_service( 'siteverification' );
+	}
```

Comment:
I'd prefer to refrain from having this shortcut at this point. Something like that should be discussed in a general modules scope, we might wanna provide a straightforward solution for all.

Reply:
I know what you mean but in this case the class will always be specific to the module, so there's no way we can abstract that in a way which will provide inspectability of each service.

Reply:
I'm open to other ideas here but this is a clean and easy solution that is easy to change later if we want.

Reply:
I think the most significant advantage a wrapper like this has right now is that the chance for typos is reduced. I think rather than introducing a new function, we should introduce a class constant for that though.

Having a function doesn't make it less to write, but on the flipside it makes following the code harder because it's one more hierarchy level to go through.

Reply:
Just to make sure we're on the same page here, the method is intended to solve this problem:

Using `get_service` loses all inspectability beyond the base `Google_Service` class.
![image](https://user-images.githubusercontent.com/1621608/64427109-0dc43380-d0b9-11e9-80c1-201b5c812a97.png)

Using the helper method preserves the full chain because it is aware of the specific type returned.
![image](https://user-images.githubusercontent.com/1621608/64427185-42d08600-d0b9-11e9-89ab-7134ef3237b9.png)
![image](https://user-images.githubusercontent.com/1621608/64427307-9e027880-d0b9-11e9-8077-229f789f4a88.png)

We can work around this with a `/* @var .. */` inline annotation but this approach seemed cleaner to me.

I'm happy to update it to be whichever way, I just wanted to make sure you understood the intent here.

Reply:
Okay that's a fair point. I'm not a fan of creating functions just to make autocompletion work though. Typically, if there is a need for this, it indicates to me that the underlying class infrastructure is not very well designed (a subclass should respect the public "interface" defined by the base class).

In this case though, we don't control the API in question, and the reason it violates the principle is because it transforms API routes into method names. Which IMO could have been solved another way too, but anyway.

This is my way of saying "I don't like it, but we can keep it for the purpose you mentioned".

One thing though: This should return just the `\Google_Service_SiteVerification`, because that's basically the reason the method exists, and it would never return something else (and it's also compatible with `Google_Service`). Also, can you add a brief description to the return value? Maybe just "The Site Verification API service." or similar.